scripts/wsrep_sst_clone.sh

```diff
@@ -358,12 +358,14 @@ setup_clone_plugin()
             WSREP_OFF="" # joiner does not have replication enabled
         fi
         $MYSQL_ACLIENT -e "${WSREP_OFF}INSTALL PLUGIN CLONE SONAME 'mysql_clone.so';"
-        CLEANUP_CLONE_PLUGIN="yes"
+        if [ "$ROLE" == "donor" ]; then
```

Comment:
It is not consistent across the script how we do the comparison. Sometimes it is `==` and sometimes it it `=` (just 4 lines above). 
`=` is posix compatible and should be used with single brackets `[...]`
`==` is bash only compatible and should be used with double brackets `[[...]]`

Would it be possible to have it consistent?

Reply:
will review all in a different branch/pr